android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -137,6 +141,10 @@ private static void throwEncodeException(JSONException e) {
     throw new RuntimeException("Could not encode JSON", e);
   }
 
+  private static void deleteFile(Context context, String fileName) {
+    context.deleteFile(fileName);
```

Comment:
I don't know, but it's probably not all that important...are you sure the filename you are using is even the right filename here?  If you have to manually determine the file name, then it's probably not the right way of doing it.